### PR TITLE
Fix tests userlist

### PR DESCRIPTION
**Pull request by ellismcdougald**
Fixes tests after UserList changes during meeting Wednesday. Mocks out UserList database interactions.

### CODE REVIEW

File: LuckyDragon/app/src/main/java/com/example/luckydragon/Models/Event.java [hunk 1]
```diff
@@ -106,6 +106,11 @@ public String toString12h() {
 
     private List<Location> waitlistLocations = new ArrayList<>();
 
+    private ArrayList<User> waitlistUsers = new ArrayList<>();
```
Comment (ahmedsiad) [inline LuckyDragon/app/src/main/java/com/example/luckydragon/Models/Event.java:109]: This code was removed in Matthew's PR, it shouldn't be here. I think your PR is still a bit messed up, it should only change stuff in the test files right?

**Review by ahmedsiad — CHANGES_REQUESTED** [review]
Can you fix the changes to non-tests before merging this?

File: LuckyDragon/app/src/main/java/com/example/luckydragon/Models/Event.java (hunk 1 above)
Comment (ellismcdougald) [reply LuckyDragon/app/src/main/java/com/example/luckydragon/Models/Event.java]: Should be fixed

File: LuckyDragon/app/src/main/java/com/example/luckydragon/Models/UserList.java [hunk 2]
```diff
@@ -41,8 +39,8 @@ public UserList(FirebaseFirestore db) {
             }
             if (value != null) {
                 users.clear();
-                for (QueryDocumentSnapshot doc: value) {
-                    users.add(createUser(doc));
+                for(int i = 0; i < value.size(); i++) {
+                    users.add(createUser((QueryDocumentSnapshot) value.getDocuments().get(i)));
```
Comment (ahmedsiad) [inline LuckyDragon/app/src/main/java/com/example/luckydragon/Models/UserList.java:43]: Any reason why this loop is changed?

**Comment by ahmedsiad** [comment]
Also I merged the notification PR into main now, can you merge from main again since I am worried some changes in Event/OrganizerEventFragment could break (cause some imports are removed)

File: LuckyDragon/app/src/main/java/com/example/luckydragon/Models/UserList.java (hunk 2 above)
Comment (ellismcdougald) [reply LuckyDragon/app/src/main/java/com/example/luckydragon/Models/UserList.java:43]: It is easier to mock a basic for loop than one that uses iterators.

**Pull request merged**